lib/_tls_common.js

```diff
@@ -169,21 +169,18 @@ exports.translatePeerCertificate = function translatePeerCertificate(c) {
   if (!c)
     return null;
 
-  if (c.issuer) c.issuer = tls.parseCertString(c.issuer);
-  if (c.issuerCertificate && c.issuerCertificate !== c) {
+  if (c.issuer != null) c.issuer = tls.parseCertString(c.issuer);
```

Comment:
I'm not sure I understand your question.  The idea is to call parseCertString() for empty strings but not null or undefined.